### DISCUSSION TITLE
Funds were not transferred from the exchange to the Chia wallet

### DISCUSSION THREAD

**Discussion by Saxus89** [Chia-Network/chia-blockchain#14038] [Support]
### Discussed in https://github.com/Chia-Network/chia-blockchain/discussions/14010

<div type='discussions-op-text'>

<sup>Originally posted by **Saxus89** November 27, 2022</sup>
Hello. 09/14/2022 withdrew 2.08 xch from the Okx exchange to the Chia wallet. The coins have not yet been received, the Okx platform says that the transaction has been completed, but there is no information about the transaction on the Chia wallet itself, the coin count is also 0. I took the transfer address from the one specified in the wallet. Is it possible to understand this situation? I apologize in advance for the online translation.</div>

**Comment by maxjay** [comment]
Contact OKX, they handle the withdrawals, not Chia Inc.
